unwind/src/lib.rs

```diff
@@ -238,6 +238,9 @@ impl RegNum {
     pub const SP: RegNum = RegNum(UNW_REG_SP);
 }
 
+#[cfg(any(target_arch = "x86_64", doc))]
```

Comment:
Will generating docs on another architecture actually work given that the unwind_sys constants these refer to won't exist?

Reply:
Yes, you are right. Removing `doc` then.